tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ModuleProjectNameValidationService.java

```diff
@@ -79,7 +79,7 @@ else if( isInvalidProjectName( op ) )
             {
                 retval = Status.createErrorStatus( "A project with that name already exists." );
             }
-            else if( isMavenProject( op ) && ! isValidMavenProjectName( currentProjectName ) )
+            else if( isGradleProject( op ) && ! isValidProjectName( currentProjectName ) )
```

Comment:
Don't add "gradle" code in project.core.  This validation should be delegated to ProjectProvider interface.